Assets/Scripts/KeyBindings/KeyBindings.cs

```diff
@@ -12,16 +12,19 @@ public static class KeyBindings
 {
     private static Dictionary<String, KeyCode> bindings = new Dictionary<string, KeyCode>();
     static KeyBindings(){
-        bindings.Add("player1_move_up", KeyCode.W);
-        bindings.Add("player1_move_left", KeyCode.A);
-        bindings.Add("player1_move_right", KeyCode.D);
-        bindings.Add("player1_move_down", KeyCode.S);
-        bindings.Add("player1_shoot_up", KeyCode.UpArrow);
-        bindings.Add("player1_shoot_left", KeyCode.LeftArrow);
-        bindings.Add("player1_shoot_right", KeyCode.RightArrow);
-        bindings.Add("player1_shoot_down", KeyCode.DownArrow);
-        bindings.Add("pause_button", KeyCode.P);
-        bindings.Add("cooldown_action", KeyCode.Space);
+        if (!StoreKeyBindings.SetDefaultBindings())
+        {
+            bindings.Add("player1_move_up", KeyCode.W);
+            bindings.Add("player1_move_left", KeyCode.A);
+            bindings.Add("player1_move_right", KeyCode.D);
+            bindings.Add("player1_move_down", KeyCode.S);
```

Comment:
Isn't there a default config containing the key bindings?
Then, you should be able to define them in the config file
and load them into this collection instead of redeclaring them

Reply:
will be done, though it doesnt make sense to remove this backup yet, as directly editing the jsons for further improvements is way worse than maintaining them here